### PR TITLE
Update texts

### PR DESCRIPTION
**Pull request by raivop**

### CODE REVIEW

File: grails-app/i18n/messages_et.properties [hunk 1]
```diff
@@ -21,10 +21,10 @@ general.speciesLists = Liiginimekirjad
 general.suppliedName = Antud nimi
 
 public.speciesLists.description = \
-    Tutvu liigirühma, kaitsealuste liikide nimestiku või mõne muu põneva liiginimekirjaga. Nimekirjad on koostatud \
-    taksonoomia ekspertide poolt ja täienevad aja jooksul. Kõik liiginimekirjad on seotud \
-    <a href="https://plutof.ut.ee" target="_blank">PlutoF</a> taksonoomiaga. Teile huvipakkuva liiginimekirja \
-    soovitamiseks võtke meiega ühendust.
+    Tutvu aasta loomade või taimede, liigirühma, kaitsealuste liikide nimestiku või mõne muu põneva liiginimekirjaga. \
+    Nimekirjad täienevad aja jooksul. Kõik liiginimekirjad on seotud \
+    <a href="https://plutof.ut.ee" target="_blank">PlutoF</> \
```
Comment (timo-piirmann) [inline grails-app/i18n/messages_et.properties]: Come on people. Why did you remove the "a".

Comment (alarku07) [reply grails-app/i18n/messages_et.properties]: js will add it later
